### PR TITLE
Show difficulty group data in Frontend

### PR DESCRIPTION
**Pull request by kathahuth**
- init static markup
- add global design
- add difficulty group boards
- add difficulty group board item

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import AppHeader from "./components/AppHeader";
 import DifficultyBoards from "./components/DifficultyBoards";
 import NavigationBar from "./components/NavigationBar";
 import PageLayout from "./components/PageLayout";
+import {useEffect, useState} from 'react'
+import {getDifficultyGroups} from "./services/MongoDbApiService";
```
Comment (fabianschmauder) [inline frontend/src/App.js]:
```suggestion
```
They are unused

File: frontend/src/pages/GroupDetails.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import {useEffect, useState} from "react";
+import {getDifficultyGroups} from "../services/MongoDbApiService";
+
+export default function GroupDetails(){
+
+    const [groupData, setGroupData] = useState([])
+
+    useEffect(() => {
+        getDifficultyGroups().then(setGroupData)
+    }, [])
```
Comment (fabianschmauder) [inline frontend/src/pages/GroupDetails.js]: you should return some JSX here ;)

Comment (fabianschmauder) [reply frontend/src/pages/GroupDetails.js]: The component is not used so i would remove it / not merge it into main ;)

**Review by fabianschmauder — COMMENTED** [review]
just some unused stuff ;) 

LGTM

File: frontend/src/components/BoardItem.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  border-radius: 10px;
+  box-shadow: 0 0 8px 3px #2f3235;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: var(--secondary-color);
+`
```
Comment (christophersiem) [inline frontend/src/components/BoardItem.js:22]:
```suggestion
import styled from "styled-components/macro";

export default styled.div`
  border: 1px solid var(--secondary-color);
  border-radius: 10px;
  box-shadow: 0 0 8px 3px #2f3235;
  margin-top: 8px;
  margin-bottom: 8px;
  padding: 20px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  background: var(--secondary-color);
`
```

File: frontend/src/pages/GroupPage.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    const [difficulties, setDifficulties] = useState([])
+
+    useEffect(() => {
+        getDifficultyGroups()
+            .then(setDifficulties)
+            .catch((error) => console.error(error))
+    }, [])
+
+    return (
+        <>
```
Comment (christophersiem) [inline frontend/src/pages/GroupPage.js:16]: don't need those fragments
```suggestion
       
```

File: frontend/src/services/MongoDbApiService.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const groupsUrl = ' /api/difficulty'
```
Comment (christophersiem) [inline frontend/src/services/MongoDbApiService.js:3]:
```suggestion
const groupsUrl = '/api/difficulty'
```

**Review by christophersiem — APPROVED** [review]
nice 🤘🏽 left some comments

File: frontend/src/components/BoardItem.js (hunk 3 above)
Comment (kathahuth) [reply frontend/src/components/BoardItem.js:22]: need this for later, to make the BoardItem clickable ;)

**Pull request merged**